docs_src/security/SeedingServiceSecrets.md

```diff
@@ -0,0 +1,118 @@
+# Seeding Service Secrets
+
+!!! edgey "EdgeX 2.1"
+    New for EdgeX 2.1 is the ability to seed service specific secrets during the service's start-up. 
+
+All EdgeX services now have the capability to specify a JSON file that contains the service's secrets which are seeded into the service's `SecretStore` during service start-up. This allows the secrets to be present in the service's `SecretStore` when the service needs to use them.
+
+!!! note
+    The service must already have a `SecretStore` configured. This is done by default for the Core/Support services. See [Configure the service's Secret Store](../Ch-Configuring-Add-On-Services/#configure-the-services-secret-store-to-use) section for details for add-on App and Device services 
+
+
+
+## Secrets File
+
+The new `SecretsFile` setting on the `SecretStore` configuration allows the service to specify the fully-qualified path to the location of the service's secrets file. Normally this setting is left blank when a service has no secrets to be seeded.
+
+!!! example "Example - Setting SecretsFilePath in TOML"

```

Comment:
`SecretsFilePath` should be `SecretsFile` to match the name in the example.

Reply:
fixed.